File: gateOne/DaySix/python/taskOne.py
import datetime

def getYears(birth_date, today):

	birth_date = datetime.datetime().("2022-12-11")

	today = datetime.datetime.now()

	year = 60 * 60 * 24 * 365

	years = (today - birth_date) / year

	return years


result = getYears("2022-12-11" , datetime.datetime.now())

print(result)
	

	

File: gateOne/DaySix/javascript/taskone.js
function getYears(fewdays, today)  {
    let Oldday = new Date("December 11, 2022 23:15:20");

    let Today = new Date();
    let todays = Today.getTime();
    let fewday = Oldday.getTime();
    let year = 1000 * 60 * 60 * 24 * 365;
    let years = (todays - fewday) / year;
    return years;

}
 
let result = getYears("December 11, 2022 23:15:20", new Date());

console.log(result);